release-notes/release-notes57.md

```diff
@@ -37,7 +37,12 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### Improved behavior when using `lists` plugin on mixed content
+
+Improved behavior when using the `lists` plugin to add lists to selections containing:
```

Comment:
```suggestion
The `lists` plugin has been updated to improve the handling of content containing:
```

Reply:
Trying to make it more of a complete sentence? It reads really weird at the moment. feels too much like a changelog entry.